onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -314,6 +314,32 @@ void RegisterBertSchemas() {
         updateOutputShape(ctx, 1, mask_index_shape);
       });
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(MaskIndex)
```

Comment:
> ONNX_CONTRIB_OPERATOR_SCHEMA(MaskIndex) [](start = 1, length = 40)

Will you split mask input out of embed_layer_norm

Reply:
Yes. In another PR.